protostar/commands/init/project_creator/_project_creator.py

```diff
@@ -31,28 +31,28 @@ def copy_template(self, cairo_version: CairoVersion, project_root_path: Path):
                 f"Folder or file named {project_root_path.name} already exists. Choose different project name."
             ) from ex_file_exists
 
-    def save_protostar_toml(
-        self, project_root_path: Path, cairo_version: CairoVersion
+    def save_protostar_toml(self, project_root_path: Path) -> None:
```

Comment:
[nitpick]: I think this method name doesn't reflect what it does anymore 😬

Reply:
Will refactor this a bit.